# Reward collecetd at the end of interval
float32 reward
# End of time interval for which the rewrad was collected , in epoch seconds
float32 time 